src/Repositories/AssetRepository.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Bazar\Repositories;
+
+class AssetRepository extends Repository
+{
+    /**
+     * Register a new asset.
+     *
+     * @param  string  $name
+     * @param  string  $path
+     * @param  string  $type
```

Comment:
Please be explicit!

```suggestion
     * @param  string  $padthai
```

🥢